code/controllers/subsystems/processing/vueui.dm

```diff
@@ -222,25 +227,49 @@ Byond Vue UI framework's management subsystem
 	if (!LAZYLEN(open_uis[old_object_key]))
 		open_uis -= old_object_key
 
-/datum/controller/subsystem/processing/vueui/proc/get_html_theme_header()
-	return {"<meta http-equiv="X-UA-Compatible" content="IE=edge"><link rel="stylesheet" type="text/css" href="vueui.css">"}
+/datum/controller/subsystem/processing/vueui/proc/get_html_theme(var/mob/user)
+	var/client/cl = null
+	if(istype(user))
+		cl = user.client
+	else
+		if(istype(user, /client))
+			cl = user
+	if(!cl)
+		return
+	var/style = cl.prefs.html_UI_style
+	if(!(style in available_html_themes))
+		style = "Nano"
+	return available_html_themes[style]
 
 /datum/controller/subsystem/processing/vueui/proc/get_html_theme_class(var/mob/user)
-	if(user.client)
-		var/style = user.client.prefs.html_UI_style
-		if(!(style in available_html_themes))
-			style = "Nano"
-		var/list/theme = available_html_themes[style]
-		var/class = ""
-		class += "[theme["class"]]"
-		if(theme["type"] == THEME_TYPE_DARK)
-			class += " dark-theme"
-		return class
-	return ""
-
-/datum/controller/subsystem/processing/vueui/proc/send_theme_resources(var/mob/user)
-	if(user.client)
-		var/datum/asset/assets = get_asset_datum(/datum/asset/simple/vueui_theming)
-		assets.send(user.client)
+	var/list/theme = get_html_theme(user)
+	if(!theme)
+		return FALLBACK_HTML_THEME
+	var/class = ""
+	class += "[theme["class"]]"
+	if(theme["type"] == THEME_TYPE_DARK)
+		class += " dark-theme"
+	return class
+
+/proc/send_theme_resources(var/user)
+#ifdef UIDEBUG
+	user << browse_rsc(file("vueui/dist/app.js"), "vueui.js")
+	user << browse_rsc(file("vueui/dist/app.css"), "vueui.css")
+#else
+	var/datum/asset/assets = get_asset_datum(/datum/asset/simple/vueui)
```

Comment:
You can condense these two lines into:
`simple_asset_ensure_is_sent(user, /datum/asset/simple/vueui)`